furious/__init__.py

```diff
@@ -13,3 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+import logging
+
+import webapp2
```

Comment:
Doing this import at the top-level is causing some consumers' tests to have a :badtime:. I suggest moving the imports to within `csrf_check`

Reply:
https://github.com/Workiva/furious/pull/189